samples/tab-channel-group-config-page-auth/csharp/ConfigTabAuthentication/Views/Authentication/SimpleLogOut.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿
+<!DOCTYPE html>
```

Comment:
Could you please check the logout implemented here? https://github.com/OfficeDev/microsoft-teams-sample-connector-csharp/blob/master/TeamsToDoAppConnector/Views/Connector/SetupAuth.cshtml#L121

Reply:
We have implemented the logout differently